### DISCUSSION TITLE
Running pyfluent in GitHub workflow fails

### DISCUSSION THREAD

**Discussion by vgelbgras** [ansys/pyfluent#2810] [Q&A]
Hi guys

I am trying to enable end-to-end tests using pyfluent.
Locally the tests works.
Through a GitHub workflow on the same machine (self-hosted Windows 10), they failed with: failed to connect to all addresses
I tried the suggestion from from [#1894](https://github.com/ansys/pyfluent/discussions/1894), it did help.

Any suggestions to debug?

**Comment by raph-luc** [comment]
Can you provide more details? Do you have a link to these GitHub actions that are failing that we can look at?

**Reply by vgelbgras** [reply]
Here is a run
[https://github.com/ansys-internal/glow-engine/actions/runs/9062484001](https://github.com/ansys-internal/glow-engine/actions/runs/9062484001)

Errors are similar to: failed to connect to all addresses; last error: UNAVAILABLE: ipv4:127.0.0.1:56802: Connection refused
The port is not always the same

The tests are using pyfluent through glow-engine. Locally they takes around 40seconds each.

In the pyfluent log I see only

```
2024-05-13 14:26:44,088 pyfluent.networking   DEBUG    GRPC_TRACE: RPC = /grpc.health.v1.Health/Check, request = {}
2024-05-13 14:28:01,406 pyfluent.networking   DEBUG    GRPC_TRACE: RPC = /grpc.health.v1.Health/Check, request = {}
2024-05-13 14:29:24,201 pyfluent.networking   DEBUG    GRPC_TRACE: RPC = /grpc.health.v1.Health/Check, request = {}
```

**Reply by vgelbgras** [reply]
On linux (Ubuntu-20.04), the same tests are passing [https://github.com/ansys-internal/glow-engine/actions/runs/9075177063/job/24935285128](https://github.com/ansys-internal/glow-engine/actions/runs/9075177063/job/24935285128)

**Reply by raph-luc** [reply]
FYI I have created an issue tracker for this: https://github.com/ansys/pyfluent/issues/2848
